pynest/nest/tests/test_aeif_lsodar.py

```diff
@@ -0,0 +1,246 @@
+# -*- coding: utf-8 -*-
+#
+# test_aeif_lsodar.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+import numpy as np
+import unittest
+import nest
+
+"""
+Comparing the new implementations the aeif models to the reference solution
+obrained using the LSODAR solver (see
+``doc/model_details/aeif_models_implementation.ipynb``).
+
+The reference solution is stored in ``test_aeif_psc_lsodar.dat`` and was
+generated using the same dictionary of parameters, the data is then downsampled
+to keep one value every 0.01 ms and compare with the NEST simulation.
+
+The new implementation binds V_m to be smaller than 0.
+
+Rationale of the test: all models should be close to the reference LSODAR when
+submitted to the same excitatory current.
+
+Details:
+  The models are compared and we assess that the difference is smaller than a
+  given tolerance.
+"""
```

Comment:
Yes, I'm currently doing it; however I don't know which path I need to provide to load the .dat file on Travis: does NEST provide a way to get the path to the source directory?